### ISSUE TITLE
RigidBody3D `get_contact_collider_velocity_at_position()` reports incorrect data

### ISSUE THREAD

**Issue by pvmelnichuk** [godotengine/godot#76611]
### Godot version

4.0.2.stable.official [7a0977ce2]

### System information

Ubuntu 22.04, Vulkan, RTX 2060 SUPER

### Issue description

Hello dear Godot devs and contributors,

I've faced an issue with rigid body contact reporting, get_contact_collider_velocity_at_position is basically not working at all, it reports only gravity component at that's it.

Minimum example of the issue attached.

Regards,
Pavel

### Steps to reproduce

Run attached example project and check output. 2 contacts are reported, both don't have any horizontal speed.

for idx in state.get_contact_count():
	print(Time.get_ticks_msec(), ' ', idx, ' ', state.get_contact_collider_velocity_at_position(idx))

Prints:
1403 0 (0, -0.163333, 0)
1404 1 (0, -0.163333, 0)

### Minimal reproduction project

[contact_velocity_min_example.zip](https://github.com/godotengine/godot/files/11361441/contact_velocity_min_example.zip)

**Comment by adamscott** [comment]
Could reproduce on 4.0.2, but not on 4.1.dev[9f12e7b]. 

On 4.1.dev, it prints:
```
1403 0 (0, 0, 0)
1404 1 (0, 0, 0)
```

---

On a side note, don't forget to format your code using backticks, like that:
\`\`\`
your code here
\`\`\`

---

And about that line: `print(Time.get_ticks_msec(), ' ', idx, ' ', state.get_contact_collider_velocity_at_position(idx))`, you can save yourself writing manual spaces by using `prints()` instead of `print()`. It adds automatically spaces between the arguments:

```gdscript
prints(Time.get_ticks_msec(), idx, state.get_contact_collider_velocity_at_position(idx))
# 1403 0 (0, 0, 0)
```

**Comment by adamscott** [comment]
Seems to be fixed by #58880. As it may break compatibility, it will not be fixed on 4.0.x

Closing the issue.
